Tests/SwerverTests/ActionSpec/NotAllowedActionSpec.swift

```diff
@@ -0,0 +1,23 @@
+import Quick
+import Nimble
+import Swerver
+import Foundation
+
+class NotAllowedActionSpec: QuickSpec {
+    
+    override func spec() {
+        describe("A Not Allowed action") {
+            let listOfAllowedMethods = ["GET", "POST"]
+            
+            let notAllowedAction = NotAllowedAction(methods: listOfAllowedMethods)
+            
+            it("can dispatch to response builder to build a response for a successful OPTIONS request") {
+                
+                let okResponse = notAllowedAction.execute()
+                expect(okResponse.statusCode).to(equal(StatusCode.not_allowed.rawValue))
+                expect(okResponse.headers[.contentType]).to(equal(ContentType.text.rawValue))
```

Comment:
I would not expose the underlying data structure for the headers. Instead, I would do `response.get(header: .allow)`

Reply:
will look into that

Reply:
response.get implemented. Question: currently, the formatter class is able to loop through the headers dictionary and extract each header, however since the response.get(header: ...) func is now implemented, should I change the access identifier from public to private?